### PR TITLE
Palo Alto Networks - hunting and detection enhancement 

### PR DESCRIPTION
**Pull request by karinafishman**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-7195

## Description
- Added Cortex XDR for hash search.
- Added Prisma Cloud for IP search.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/PANWComprehensiveInvestigation/ReleaseNotes/1_3_23.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+#### Playbooks
+
+##### Palo Alto Networks - Hunting And Threat Detection
+
+- Added Cortex XDR for hash search.
```
Comment (ssokolovich) [inline Packs/PANWComprehensiveInvestigation/ReleaseNotes/1_3_23.md:6]:
```suggestion
- Added a hash search enrichment using Cortex XDR.
```

File: Packs/PANWComprehensiveInvestigation/ReleaseNotes/1_3_23.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+- Added Prisma Cloud for IP search.
```
Comment (ssokolovich) [inline Packs/PANWComprehensiveInvestigation/ReleaseNotes/1_3_23.md:7]:
```suggestion
- Added an IP search enrichment using Prisma Cloud.
```

File: Packs/PANWComprehensiveInvestigation/doc_files/Palo_Alto_Networks_-_Hunting_And_Threat_Detection.png
Comment (ssokolovich) [inline Packs/PANWComprehensiveInvestigation/doc_files/Palo_Alto_Networks_-_Hunting_And_Threat_Detection.png:1]: No `else` path for the Prisma cloud ip search conditions. Please add it.

File: Packs/PANWComprehensiveInvestigation/Playbooks/Palo_Alto_Networks_-_Hunting_And_Threat_Detection_README.md
Comment (ssokolovich) [inline Packs/PANWComprehensiveInvestigation/Playbooks/Palo_Alto_Networks_-_Hunting_And_Threat_Detection_README.md:1]: missing the Prisma check in the description

File: Packs/PANWComprehensiveInvestigation/Playbooks/Palo_Alto_Networks_-_Hunting_And_Threat_Detection.yml [hunk 3]
```diff
@@ -3791,26 +3798,424 @@ tasks:
...
+    type: condition
+    task:
+      id: 14629675-a0f1-460b-8f0f-e9d91acead33
+      version: -1
+      name: Are there any relevant IP's?
+      type: condition
+      iscommand: false
+      brand: ""
+      description: ""
+    nexttasks:
```
Comment (ssokolovich) [inline Packs/PANWComprehensiveInvestigation/Playbooks/Palo_Alto_Networks_-_Hunting_And_Threat_Detection.yml:4109]: need else here

File: Packs/PANWComprehensiveInvestigation/Playbooks/Palo_Alto_Networks_-_Hunting_And_Threat_Detection.yml [hunk 4]
```diff
@@ -3791,26 +3798,424 @@ tasks:
...
+    task:
+      id: 1581d291-9a38-483e-842a-91aa3a4b7953
+      version: -1
+      name: Is Prisma Cloud enabled?
+      description: Returns 'yes' if integration brand is available. Otherwise returns 'no'
+      scriptName: IsIntegrationAvailable
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
```
Comment (ssokolovich) [inline Packs/PANWComprehensiveInvestigation/Playbooks/Palo_Alto_Networks_-_Hunting_And_Threat_Detection.yml:4040]: need `else` here

File: Packs/PANWComprehensiveInvestigation/Playbooks/Palo_Alto_Networks_-_Hunting_And_Threat_Detection.yml [hunk 5]
```diff
@@ -3791,26 +3798,424 @@ tasks:
...
+      {
+        "position": {
+          "x": -4690,
+          "y": 1350
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
```
Comment (ssokolovich) [inline Packs/PANWComprehensiveInvestigation/Playbooks/Palo_Alto_Networks_-_Hunting_And_Threat_Detection.yml:4093]:
```suggestion
    skipunavailable: true
```

File: Packs/PANWComprehensiveInvestigation/Playbooks/Palo_Alto_Networks_-_Hunting_And_Threat_Detection.yml [hunk 6]
```diff
@@ -3791,26 +3798,424 @@ tasks:
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "179":
+    id: "179"
+    taskid: d352ae69-ed2f-40c7-87e8-b8b33e48bbb4
+    type: regular
+    task:
+      id: d352ae69-ed2f-40c7-87e8-b8b33e48bbb4
+      version: -1
+      name: Set detected IP's
```
Comment (ssokolovich) [inline Packs/PANWComprehensiveInvestigation/Playbooks/Palo_Alto_Networks_-_Hunting_And_Threat_Detection.yml:3920]:
```suggestion
      name: Set Cortex XDR agent detected IP's
```

File: Packs/PANWComprehensiveInvestigation/Playbooks/Palo_Alto_Networks_-_Hunting_And_Threat_Detection.yml [hunk 7]
```diff
@@ -2,7 +2,7 @@ id: Palo Alto Networks - Hunting And Threat Detection
...
 name: Palo Alto Networks - Hunting And Threat Detection
-description: "This is a multipurpose playbook used for hunting and threat detection. The playbook receives inputs based on hashes, IP addresses, or domain names provided manually or from outputs by other playbooks. \nWith the received indicators, the playbook leverages data received by PANW products including, Strata Logging Service, Autofocus and Pan-OS to search for IP addresses, host names and users related to the provided indicators.\nThe output provided by the playbook facilitates pivoting searches for possibly affected IP addresses or users."
+description: "This is a multipurpose playbook used for hunting and threat detection. The playbook receives inputs based on hashes, IP addresses, or domain names provided manually or from outputs by other playbooks. \nWith the received indicators, the playbook leverages data received by PANW products including, Strata Logging Service, Autofocus, Prisma Cloud and Pan-OS to search for IP addresses, host names and users related to the provided indicators.\nThe output provided by the playbook facilitates pivoting searches for possibly affected IP addresses or users."
```
Comment (ssokolovich) [inline Packs/PANWComprehensiveInvestigation/Playbooks/Palo_Alto_Networks_-_Hunting_And_Threat_Detection.yml:5]:
```suggestion
description: "This is a multipurpose playbook used for hunting and threat detection. The playbook receives inputs based on hashes, IP addresses, or domain names provided manually or from outputs by other playbooks. \nWith the received indicators, the playbook leverages data received by PANW products including, Strata Logging Service, Autofocus, Prisma Cloud ,Cortex XDR and Pan-OS to search for IP addresses, host names and users related to the provided indicators.\nThe output provided by the playbook facilitates pivoting searches for possibly affected IP addresses or users."
```

**Review by ssokolovich — APPROVED** [review]

File: Packs/PANWComprehensiveInvestigation/Playbooks/Palo_Alto_Networks_-_Hunting_And_Threat_Detection_README.md [hunk 8]
```diff
@@ -34,10 +43,11 @@ This playbook does not use any integrations.
...
-| InternalDomainName | The organizations internal domain name. This is provided for the script IsInternalHostName that checks if the detected host names are internal or external if the hosts contain the internal domains suffix. For example demisto.com. If there is more than one domain, use the \| character to separate values such as \(demisto.com\|test.com\) |  | Optional |
+| InternalDomainName | The organization's internal domain name. This is provided for the script IsInternalHostName that checks if the detected host names are internal or external if the hosts contain the internal domains suffix. For example demisto.com. If there is more than one domain, use the \| character to separate values such as \(demisto.com\|test.com\) |  | Optional |
 | InternalHostRegex | This is provided for the script IsInternalHostName that checks if the detected host names are internal or external. If the hosts match the organization's naming convention. For example the host testpc1 will have the following regex \\w\{6\}\\d\{1\} |  | Optional |
```
Comment (tschanfeld) [inline Packs/PANWComprehensiveInvestigation/Playbooks/Palo_Alto_Networks_-_Hunting_And_Threat_Detection_README.md:47]: @karinafishman this is where the partial sentence is 'If the hosts match the organization's naming convention' - it's not something new, was already in the file

**Pull request merged**